web/src/app/core/services/http.service.ts

```diff
@@ -0,0 +1,46 @@
+// Core modules
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+
+// Rxjs operators
+import { throwError, Observable } from 'rxjs';
+import { catchError } from 'rxjs/operators';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class HttpService {
+
+  /**
+   * Constructor
+   *
+   * @param 'httpClient'
+   */
+  constructor(
+    protected httpClient: HttpClient
+  ) { }
+
+  /**
+   * Standardized RMS GET request
+   *
+   * @param 'uri'
+   * @param 'options'
+   */
+  get(uri: string): any {
+    return this.httpClient
+      .get(uri, { responseType: 'text' }).pipe(catchError(
+        this.handleError
+      ));
+  }
+
+  /**
+   * Handle general errors from the API
+   *
+   * @param 'err'
+   * @returns returns
+   */
+  private handleError(err: any): Observable<Error> {
+
+    return throwError(err);
+  }
+}
```

Comment:
This new general service is not needed, please use `HttpClient` in the existing repository service `web/src/app/core/services/repository.service.ts`